api/Dockerfile

```diff
@@ -23,6 +24,6 @@ COPY --from=builder ${VIRTUAL_ENV} ${VIRTUAL_ENV}
 
 COPY . ./
 
-RUN manage.py collectstatic
+RUN python ./manage.py collectstatic
```

Comment:
this doesn't solve the problem for me, I made [commit about it on my branch](https://github.com/knmlprz/ChatKNML/pull/97/commits/b64f6b4ad0cfac238232b3f32d7c1d444f92b6ee), this version works for me

Reply:
Jaki ci błąd wywala? Czyżby coś takiego?
![image](https://github.com/knmlprz/ChatKNML/assets/32245772/4aad3412-48f1-4528-a4a3-de1e3d1413e6)